### DISCUSSION TITLE
How to run pkg with  a custom built binary

### DISCUSSION THREAD

**Discussion by faulpeltz** [yao-pkg/pkg-fetch#41] [Q&A] [answered]
I've been playing around with creating a new patch for node 20.17.0 and I managed to build a (linux x64) binary which seems to work.

Whats the quickest way to test this binary with an existing pkg config?

**Comment by robertsLando** [comment, marked as answer]
Here you can find an example: https://github.com/yao-pkg/pkg-binaries#usage

**Reply by faulpeltz** [reply]
Thanks, that worked, would you like a PR for 20.17 ?

**Reply by robertsLando** [reply]
@faulpeltz I would like to firstly focus on fixing mac os build and windows build as well.

See #37 and #40

**Reply by faulpeltz** [reply]
Sure - the changes are minimal anyway

**Reply by robertsLando** [reply]
If you would like to contribute that is what is blocking new versions
